doc/releases/changelog-dev.md

```diff
@@ -414,49 +437,32 @@ of operators. [(#2622)](https://github.com/PennyLaneAI/pennylane/pull/2622)
 * Add trivial behaviour logic to `qml.operation.expand_matrix`.
```

Comment:
Not sure what this is 🤔. New feature? Improvement? Needs more context

Reply:
I think it is an improvement of the existing `expand`, let's keep it here.

Reply:
No I don't think it needs to be advertise with an example here, the change is minimal.

Reply:
Okay!